components/donate-button.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {FormattedMessage, IntlMixin, FormattedNumber} from 'react-intl';
+
+module.exports = React.createClass({
```

Comment:
Shared by sequential and simple paypal.

Not sure about the name. It's actually not much of a button, just kinda goes into a button.